Lib9c/BlockHelper.cs

```diff
@@ -36,18 +37,19 @@ public static class BlockHelper
             // FIXME Must use a separate key for the mainnet.
             var minterKey = new PrivateKey();
             var ncg = new Currency("NCG", 2, minterKey.ToAddress());
+            activatedAccounts = activatedAccounts ?? ImmutableHashSet<Address>.Empty;
             var initialStatesAction = new InitializeStates
             {
                 RankingState = new RankingState(),
                 ShopState = new ShopState(),
                 TableSheets = (Dictionary<string, string>) tableSheets,
                 GameConfigState = gameConfigState,
                 RedeemCodeState = new RedeemCodeState(redeemCodeListSheet),
-                AdminAddressState = new AdminState(adminAddress, 1500000),
+                AdminAddressState = adminState,
                 ActivatedAccountsState = new ActivatedAccountsState(
                     isActivateAdminAddress
-                    ? ImmutableHashSet<Address>.Empty.Add(adminAddress)
-                    : ImmutableHashSet<Address>.Empty),
+                    ? activatedAccounts.Add(adminState.AdminAddress)
+                    : activatedAccounts),
```

Comment:
Is there a case where we need the option to activate the rest of the addresses except for the adminstrator?

Reply:
I guess... nope maybe. 😅